src/handlers/shared/start.ts

```diff
@@ -33,6 +36,21 @@ export async function start(context: Context, issue: Context["payload"]["issue"]
     logger.error("Error while getting commit hash", { error: e as Error });
   }
 
+  // check max assigned issues
+  const openedPullRequests = await getAvailableOpenedPullRequests(context, sender.login);
```

Comment:
these checks can be removed since we process checks for each user now in `handleTaskLimitChecks()`